### ISSUE TITLE
Ban from anidb (555)

### ISSUE THREAD

**Issue by enlait** [japanesemediamanager/jmmserver#148]
I've been getting these quite for some time already: getting data from anidb results in "Paused due to BAN from AniDB (UDP)"

In log: 
[2015-05-25 20:39:24:741] Info|AniDBCommand_Login.Process => AniDBUDPCommand.ProcessCommand => JMMService.LogToDatabase AniDB UDP - UDP_RESPONSE_TRUNC in 593,5952ms - 555 BANNED Excessive"

I tried waiting for days and weeks between launching JMM again, but the issue persist.
I can use anidb website with no issues, and have no notifications of misconduct.

Restarting anidb queue results in these messages in log for each action:

Error|AniDBHelper.Login => AniDBCommand_Login.Process => AniDBUDPCommand.ProcessCommand System.Net.Sockets.SocketException (0x80004005): A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
   at System.Net.Sockets.Socket.ReceiveFrom(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags, EndPoint& remoteEP)
   at System.Net.Sockets.Socket.ReceiveFrom(Byte[] buffer, EndPoint& remoteEP)
   at AniDBAPI.Commands.AniDBUDPCommand.ProcessCommand(Socket& soUDP, IPEndPoint& remoteIpEndPoint, String sessionID, Encoding enc) in c:\Projects\GitHub\JMMServer\JMMServer\AniDB_API\Commands\AniDBUDPCommand.cs:line 123

JMM server version reports 3.4.2.19
It updated today, but was already 3.4.2.19 before update.

What should I do?
I'd like to get this to work, JMM was the only thing that made managing my collection feasible.

**Comment by jmediamanager** [comment]
Hi, this can happen every now and then with AniDB.
The best thing to do is raise an issue on the AniDB forum and get the ban removed.
I have discussed this with Ommina in the past before.

Raise the issue here
http://anidb.net/perl-bin/animedb.pl?show=threads&id=19

To get a good response make sure you provide the following details
- You IP Address
- Ban is from the UDP API, not the HTTP API
- The Client you are using : JMM Server
- The above error message "555 BANNED Excessive"

**Comment by enlait** [comment]
Ommina said "excessive" = "wrong password" in my case. I would have never guessed.
So it's starting to make some sense now. I think thats what happened:
- when I became unable to sync stuff to anidb (around February?) I found issue #132 discussion, suggesting to update. I re-read logs, I had 504 at that time. Since autoupdating didn't work, I reinstalled JMM.
- I seem to fk up entering password, so I still got rejected from anidb, but for another reason, and I didn't notice
- error message did not relate to actual problem

Even after changing password, it took me a few restarts until things start working - I was getting timeout errors like the one in OP. But this is possibly related to my ISP failing me these days with pings of over 1000ms to api,anidb.info (in fact, to anywhere, including network gateway the next hop from my router).

Thank you for your help!
